backend/src/api/routes/post_paper.ts

```diff
@@ -37,4 +56,14 @@ export default class PostPaper implements ApiRoute {
         });
     }
 
+    createEmbeddingText(paper: Paper): string {
+        return `Title: ${paper.title} 
+        Abstract: 
+        ${paper.abstract}
+
+        Subjects: ${paper.subjects.join(', ')}
+        
+        Authors: ${paper.authors.join(', ')}`;
+    }
```

Comment:
This part is experimental and may change once I see how the search behaves with it